dev/com.ibm.ws.microprofile.faulttolerance.2.0/src/com/ibm/ws/microprofile/faulttolerance20/impl/AsyncExecutor.java

```diff
@@ -143,8 +143,7 @@ private void enqueueAttempt(AsyncExecutionContextImpl<W> executionContext) {
 
         timeout.start();
 
-        ExecutionReference ref = bulkhead.submit(handleExceptions(() -> runExecutionAttempt(attemptContext), attemptContext, executionContext),
-                                                 getExceptionHandler(attemptContext));
+        ExecutionReference ref = bulkhead.submit((reservation) -> runExecutionAttempt(attemptContext, reservation), getExceptionHandler(attemptContext));
```

Comment:
Removal of `handleExceptions()` is due to `runExecutionAttempt` now doing its own exception handling, since it has slightly more to do to ensure that the bulkhead is always released.